### PR TITLE
Build full website with react router

### PR DESCRIPTION
**Pull request by utsargo**
### Description

- Add react-router
- Add Homepage
- Add Quote page

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,8 +1,26 @@
...
+  <main>
+    <nav className="navigation">
+      <h1>Math Magicians</h1>
+      <ul>
+        <Link to="/">Home</Link>
+        <Link to="/calculator">Calculator</Link>
+        <Link to="/quote">Quote</Link>
+      </ul>
+      <hr />
+    </nav>
```
Comment (Owusu-Desmond) [inline src/App.js]: - [ ] From the project requirement, the general layout of this app should match the given wireframes in the [project requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_full_website.md). But comparing the given wireframes to your project layout, your navigation bar does not match the given wireframe. As you can see below, the `Math Magicians` text should lie on the right and the navigation links should lie on the left on the same line.
### Your layout
Screenshot from 2023-01-26 07-51-54

### Wireframe
Screenshot from 2023-01-26 07-44-05

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @utsargo , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the setup of the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js [hunk 2]
```diff
@@ -12,32 +12,35 @@ const Calculator = () => {
...
+          <button type="button" className="grid-button number" onClick={clickHandle}>5</button>
+          <button type="button" className="grid-button number" onClick={clickHandle}>6</button>
+          <button type="button" className="grid-button operator column-4" onClick={clickHandle}>-</button>
+          <button type="button" className="grid-button number" onClick={clickHandle}>1</button>
+          <button type="button" className="grid-button number" onClick={clickHandle}>2</button>
+          <button type="button" className="grid-button number" onClick={clickHandle}>3</button>
+          <button type="button" className="grid-button operator column-4" onClick={clickHandle}>+</button>
+          <button type="button" className="grid-button number" id="zero" onClick={clickHandle}>0</button>
+          <button type="button" className="grid-button number" onClick={clickHandle}>.</button>
+          <button type="button" className="grid-button operator column-4" onClick={clickHandle}>=</button>
```
Comment (bobb-Rob) [inline src/components/Calculator.js:42]: - [ ] Great work you have done so far on this project. However, this milestone was about testing your application using Jest and React testing Library and I have not seen any test code in a `_.test.js` file. Please see [project requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_tests.md). 
* **The test should cover:** 
  - All the functionalities & scenarios of the `operate.js` and `calculate.js` modules should be covered using Jest.
  - There should be the usage of Jest snapshots for testing all the components.
  - There should be the usage of React Testing Library to simulate user interaction.

Please implement this a re-submit for review.

**Review by bobb-Rob — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️
Hi @utsargo, 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you are almost there!  😃 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Highlights
✅ No Linter error
✅ Correct use of GitHub flow
✅ Very descriptive Pull Request(PR)
✅ Neat Commit messages

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 2 above)
Comment (utsargo) [reply src/components/Calculator.js:42]: @bobb-Rob 
This submission was done by mistake. It's not for the test. I'll create another PR for that. Thank you.

**Review by codepantha — APPROVED** [review]
# STATUS: APPROVED 🟢 ✔️ 

## Hi @utsargo 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**